File: PHP/Newton´s  Laws exercise/Laws/FirstLaw.php
<?php
include_once 'Laws.php';

class FirstLaw extends Laws{
    
    private String $calculate;
    private String $name;

    public function __construct(int $object) {
        parent::__construct($object);
        $this->calculate="";
        $this->name="First Law";

    }

    public function getfirstLaw(): String {
        $this->calculate=($this->getObject()===0)
        ?  "The object maintains its state:" 
        :  "The object is being affected by an external force";
        return $this->calculate;
    }

    public function getExplain(): string {
        return "Every body remains in its state of rest or uniform rectilinear motion unless acted upon by an external force.";
    }

    public function getName(): string {
        return $this->name;
    }
}

File: JAVA/Newton´s  Laws exercise/com/Laws/FirstLaw.java
package com.Laws;

public class FirstLaw extends Laws {
    
    private String calculate;
    private String name;

    public FirstLaw(int object) {
        super(object);
        this.calculate = "";
        this.name = "First Law";
    }

    public String getName() {
        return this.name;
    }

    public String getFirstLaw() {
        this.calculate = (this.getObject() == 0)
            ? "The object maintains its state:" 
            : "The object is being affected by an external force";
        return this.calculate;
    }

    @Override
    public String getExplain() {
        return "Every body remains in its state of rest or uniform rectilinear motion unless acted upon by an external force.";
    }


}